src/main/java/seedu/mark/model/reminder/Reminder.java

```diff
@@ -0,0 +1,79 @@
+package seedu.mark.model.reminder;
+
+import static seedu.mark.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+import seedu.mark.model.bookmark.Bookmark;
+
+
+/**
+ * Represent a reminder in Mark.
+ */
+public class Reminder {
+
+    //temporarily
```

Comment:
```suggestion
    //TODO: temporarily; to change when confirmed
```
Was thinking that this makes it clearer as Dorcas mentioned so we don't forget this bit.

Reply:
Okay